inference-contract/docs/notebooks/templates/token-issuer.py

```diff
@@ -13,26 +13,27 @@
 # ---
 
 # %% [markdown]
-# # Token Issuer Notebook
-# This section enables customization of the token that will be minted. Edit the variables in the section below as necessary.
+# Token Issuer Notebook This section enables customization of the token that will be minted. Edit
+# the variables in the section below as necessary.
 #
 # * token_owner : the identity of the token creator
-# * token_class : the name of tokens that will be generated, this is only used to simplify local access (e.g. context file name)
+# * token_class : the name of tokens that will be generated, used to simplify local access
 # * token_description : a description of the asset associated with the minted tokens
 # * token_metadata : additional information about the token
 # * count : the number of tokens to mint for the asset
-# * service_host : the host for the eservice where tokens will be minted, this will use the default service group
+# * service_group : the service group used for PDO services
+# * guardian_host : the host where the guardian service runs
 #
-# Note that the notebook assumes that there is a key file for the identity of the form: `${keys}/${identity}_private.pem`.
-
-
+# Note that the notebook assumes that there is a key file for the identity of the form:
+# `${keys}/${identity}_private.pem`.
 # %% tags=["parameters"]
 token_owner = 'user1'
 token_class = 'mytoken'
 token_description = 'this is my token'
 token_metadata = 'created by {}'.format(token_owner)
 count = 1
-service_host = 'localhost'
+service_group = 'default'
+guardian_host = 'localhost'
```

Comment:
i think we should  paramterize guardian_host, and not hardcode to localhost. specifically, the guardian does not sit on the same node as the client.

Reply:
this is a default. it is parameterized in the notebook.